### ISSUE TITLE
[LiveLeak] Embedded YouTube videos are not detected

### ISSUE THREAD

**Issue by joepie91** [rg3/youtube-dl#7506]
- **URLs:** http://www.liveleak.com/view?i=993_1447462861, http://www.liveleak.com/view?i=cef_1447457924
- **Platform:** openSUSE 13.1, x86_64, XFCE
- **Version:** 2015.11.13

As issue title says. Issue occurs for both URLs above, only the output for one of them (using an alias) is included below.

```
sven@linux-etoq:/mnt/storage2/archive/paris-attacks-youtube/liveleak> ytdl -v http://www.liveleak.com/view?i=993_1447462861
[debug] System config: []
[debug] User config: []
[debug] Command-line args: [u'--title', u'--continue', u'--retries', u'4', u'--write-info-json', u'--write-description', u'--write-thumbnail', u'--write-annotations', u'--all-subs', u'--ignore-errors', u'--merge-output-format', u'mkv', u'-f', u'bestvideo+bestaudio/best', u'-v', u'http://www.liveleak.com/view?i=993_1447462861']
[debug] Encodings: locale UTF-8, fs UTF-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2015.11.13
[debug] Python version 2.7.6 - Linux-3.11.10-29-desktop-x86_64-with-SuSE-13.1-x86_64
[debug] exe versions: ffmpeg 2.8.2, ffprobe 2.8.2, rtmpdump 2.4
[debug] Proxy map: {}
[LiveLeak] 993_1447462861: Downloading webpage
ERROR: Unable to extract embed URL; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/youtube_dl/YoutubeDL.py", line 661, in extract_info
    ie_result = ie.extract(url)
  File "/usr/lib/python2.7/site-packages/youtube_dl/extractor/common.py", line 290, in extract
    return self._real_extract(url)
  File "/usr/lib/python2.7/site-packages/youtube_dl/extractor/liveleak.py", line 79, in _real_extract
    webpage, 'embed URL')
  File "/usr/lib/python2.7/site-packages/youtube_dl/extractor/common.py", line 584, in _search_regex
    raise RegexNotFoundError('Unable to extract %s' % _name)
RegexNotFoundError: Unable to extract embed URL; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
```

**Comment by jaimeMF** [comment]
#6869 probably fixes it.

**Comment by aeph6Ee0** [comment]
I can confirm that my patch #6869 can extract those videos. Why is it not merged yet?

**Comment by yan12125** [comment]
This is already fixed by #10688. Feel free to open a new issue if there are still problems in the latest version.
